src/test/javascript/spec/component/admin/user-management.component.spec.ts

```diff
@@ -22,15 +42,28 @@ describe('UserManagementComponent', () => {
         children: [],
     } as any as ActivatedRoute;
 
-    beforeEach(async () => {
+    beforeEach(() => {
         return TestBed.configureTestingModule({
-            imports: [TranslateModule.forRoot(), ArtemisTestModule, ArtemisAdminModule, RouterTestingModule],
+            imports: [MockModule(ReactiveFormsModule), MockModule(NgbModule), HttpClientTestingModule, MockModule(RouterTestingModule)],
             providers: [
                 {
                     provide: ActivatedRoute,
                     useValue: route,
                 },
                 { provide: AccountService, useClass: MockAccountService },
+                { provide: TranslateService, useClass: MockTranslateService },
```

Comment:
Both the translate service mock and the faIconComponentMock would be included in the ArtemisTestModule, I am not sure which solution is better, though

Reply:
I'll import the ArtemisTestModule